ouroboros-consensus-cardano/test/cardano-test/Test/Consensus/Cardano/MiniProtocol/LocalTxSubmission/Server.hs

```diff
@@ -47,27 +47,23 @@ import           Test.Tasty.HUnit (testCase, (@=?))
 import qualified Test.ThreadNet.Infra.Shelley as Shelley
 
 tests :: TestTree
-tests = testGroup "LocalTxSubmissionServer"
-    [ localServerCanProcessGoldenTransactions (stayInByron         , 0)
-    , localServerCanProcessGoldenTransactions (hardForkInto Shelley, 1)
-    , localServerCanProcessGoldenTransactions (hardForkInto Allegra, 2)
-    , localServerCanProcessGoldenTransactions (hardForkInto Mary   , 3)
-    , localServerCanProcessGoldenTransactions (hardForkInto Alonzo , 4)
-    , localServerCanProcessGoldenTransactions (hardForkInto Babbage, 5)
-    , localServerCanProcessGoldenTransactions (hardForkInto Conway,  6)
-    ]
+tests =
+    testGroup "LocalTxSubmissionServer"
+      $  [ localServerPassesRegressionTests (stayInByron         , 0) ]
```

Comment:
Hmmm, maybe this would look better if `stayInByron == hardForkInto Byron`.

Reply:
You could also use the `Show` instance for `Era` instead of `eraName` then